.github/workflows/pull_request.yml

```diff
@@ -88,3 +88,8 @@ jobs:
     name: Static SDK
     # Workaround https://github.com/nektos/act/issues/1875
     uses: apple/swift-nio/.github/workflows/static_sdk.yml@main
+
+  wasm-sdk:
+    name: WebAssembly SDK
```

Comment:
> We sadly cannot accept this PR at this time. Currently WASM is not an officially supported platform by Swift…

@FranzBusch Curious what your thoughts are about allowing wasm builds into `swif-nio`, but as `optional` actions rather than `required`? 

This would improve visibility on wasm breakages without imposing a requirement for official support. Right now, contributors are likely a bit blind to any wasm breakages. If there is a way to improve awareness without imposing extra burden on contributors, feels like a win-win.

I believe we could make it optional by adding the following config to `wasm-sdk` here.

```
continue-on-error: true
```

Reply:
cc @MaxDesiatov in case you have any insight on this as well, as an Apple employee.

Reply:
> @FranzBusch Curious what your thoughts are about allowing wasm builds into `swif-nio`, but as `optional` actions rather than `required`?

I'm fully empathetic on making sure we are not breaking WASM support. However, at this point we just can't use an externally produced toolchain here. That's why I would like to push the Platforms Steering Group on considering WASM as an official platform so that we can start producing official toolchains that we can use in our CI.

Reply:
One more data point on this topic, currently `swiftlang/swift` has at least some wasm CI visibility. It doesn't look like it is a requirement in Pull Requests. But the wasm build status for swift is reported prominently on the repository welcome page.

![CleanShot 2025-03-27 at 10 53 20](https://github.com/user-attachments/assets/6011b5ac-a5f3-4516-ad91-b8a4350f0eaa)

Reply:
6.2 and `main` snapshots now have corresponding Swift SDKs distributed on swift.org https://www.swift.org/install/